prov/efa/src/rxr/rxr_rma.c

```diff
@@ -497,8 +459,12 @@ ssize_t rxr_rma_post_write(struct rxr_ep *ep, struct rxr_tx_entry *tx_entry)
 		 */
 	}
 
-	ctrl_type = delivery_complete_requested ?
-		RXR_DC_LONGCTS_RTW_PKT : RXR_LONGCTS_RTW_PKT;
+	if (tx_entry->total_len <= max_eager_rtw_data_size) {
```

Comment:
Where is `max_eager_rtw_data_size` calculated now?

Reply:
line 441/444:

```
		max_eager_rtw_data_size = rxr_tx_entry_max_req_data_capacity(ep, tx_entry, RXR_DC_EAGER_RTW_PKT);
	} else {
		max_eager_rtw_data_size = rxr_tx_entry_max_req_data_capacity(ep, tx_entry, RXR_EAGER_RTW_PKT);
	}
```